src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,28 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("121212@gmail.com", "qwerty");
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
+        }
+        User user;
+        try {
+            user = authenticationService.login("121212@gmail.com", "qwerty");
```

Comment:
Ensure that the login credentials are correct and handle cases where `authenticationService.login` might return null or throw an exception, to avoid runtime errors.